public/src/helpers/formHelpers.jsx

```diff
@@ -21,3 +22,55 @@ export default function renderField(field) {
     </div>
   );
 }
+
+export function renderTextArea(field) {
+  const className = `form-group ${field.meta.touched && field.meta.error ? 'has-error' : ''}`;
+  return (
+    <div className={className}>
+      <label htmlFor={field.label}>{field.label}</label>
+      <textarea
+        cols="50"
+        rows="6"
+        className="form-control"
+        placeholder={field.label}
+        {...field.input}
+      />
+      <div className="text-danger">
+        {field.meta.touched ? field.meta.error : ''}
+      </div>
+    </div>
+  );
+}
+
+function renderSelectOptions(options, prop) {
+  return (
+    options.map(option =>
+      (
+        <option
+          key={option[prop]}
+          value={option.id}
+        >
+          {option[prop]}
+        </option>
+      )
+    )
+  );
+}
+
+
+export function renderDropdown(field) {
+  const className = `row form-group ${field.meta.touched && field.meta.error ? 'has-error' : ''}`;
+  return (
+    <div className={className}>
+      <label htmlFor={`dropdown-basic-${field.id}`}>{field.label}</label>
+      <select
+        id={`dropdown-basic-${field.id}`}
+        className="form-control"
+        {...field.input}
+      >
+        <option>Select Question</option>
```

Comment:
We should change  this to <option>Select {field.label}</option> so we can specify what we're selecting. Currently it's being hard coded as questions.